### PR TITLE
[KYUUBI #5583][AUTHZ] Authz support PythonLogicalOperator

### LINKED ISSUE #5583: [Improvement] Authz support PythonLogicalOperator
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/kyuubi/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

Authz support MapInPandas

### How should we improve?

_No response_

### Are you willing to submit PR?

- [X] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to improve.
- [ ] No. I cannot submit a PR at this time.

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### _Why are the changes needed?_
To close #5583 
Authz support PythonLogicalOperator

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### _Was this patch authored or co-authored using generative AI tooling?_
No

### CODE REVIEW

**Comment by yaooqinn** [comment]
Is it possible to make e2e tests？

**Comment by AngersZhuuuu** [comment]
> Is it possible to make e2e tests？

Trying

**Comment by AngersZhuuuu** [comment]
> Is it possible to make e2e tests？

Done

File: extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/TableCommands.scala [hunk 1]
```diff
@@ -600,8 +600,49 @@ object TableCommands extends CommandSpecs[TableCommandSpec] {
...
   }
 
+  val FlatMapGroupsInPandas = {
+    val cmd = "org.apache.spark.sql.catalyst.plans.logical.FlatMapGroupsInPandas"
+    TableCommandSpec(cmd, Seq.empty, QUERY, queryDescs = Seq(QueryDesc("child")))
+  }
+
+  val MapInPandas = {
+    val cmd = "org.apache.spark.sql.catalyst.plans.logical.MapInPandas"
+    TableCommandSpec(cmd, Seq.empty, QUERY, queryDescs = Seq(QueryDesc("child")))
```
Comment (yaooqinn) [inline extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/TableCommands.scala:610]: Have we already visited the child in buildQuery？

Comment (AngersZhuuuu) [reply extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/TableCommands.scala:610]: > Have we already visited the child in buildQuery？

Oh I made a mistake, pyspark's mapInPandas just uses map_partition method

Comment (yaooqinn) [reply extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/TableCommands.scala:610]: I appreciate your confirmation. Let's keep the tests if you're interested.

**Pull request closed without merging**